storage/test_data.py

```diff
@@ -40,53 +40,29 @@ def test_file_get(client):
 def test_file_put(client):
     init_storage_test()
 
-    assert not os.path.exists(os.path.join(FILE_STORE, "test.txt"))
+    assert not os.path.exists(os.path.join(FILE_STORE, "1"))
     data = "AAAAAAAAAAABBBBBBBBBBCCCCCCCCCCCCCC"
 
-    resp = client.put("/file?filename=a.txt", data=data)
+    resp = client.post("/file?filename=1", data=data)
     assert resp.status_code == 201
 
-    resp = client.put("/file?filename=../b.txt", data=data)
+    resp = client.post("/file?filename=../b.txt", data=data)
     assert resp.status_code == 400
 
-    resp = client.put("/file?filename=a.txt", data=data)
+    resp = client.post("/file?filename=1", data=data)
     assert resp.status_code == 400
 
-    resp = client.put("/file?filename=/folder/b.txt", data=data)
-    assert resp.status_code == 404
-
 
 def test_file_delete(client):
     init_storage_test()
-    fpath = os.path.join(FILE_STORE, "a.txt")
+    fpath = os.path.join(FILE_STORE, "1")
     open(fpath, "w")
 
-    assert client.get("/file?filename=a.txt").status_code == 200
+    assert client.get("/file?filename=1").status_code == 200
 
-    assert client.delete("/file?filename=a.txt").status_code == 200
+    assert client.delete("/file?filename=1").status_code == 200
     assert not os.path.exists(fpath)
 
 
-def test_mkdir(client):
-    init_storage_test()
-
-    resp = client.put("/directory?name=a")
-
-    assert resp.status_code == 201
-    assert os.path.exists(os.path.join(FILE_STORE, "a"))
-
-    resp = client.put("/directory?name=a/b/c")
-    assert resp.status_code == 400
-
-
-def test_delete_dir(client):
-    init_storage_test()
-    dirname = os.path.join(FILE_STORE, "a")
-    os.mkdir(dirname)
-
-    resp = client.delete("/directory?name=a")
-    assert resp.status_code == 200
-    assert not os.path.exists(dirname)
-
 def test_filesystem(client):
-    pass
\ No newline at end of file
+    pass
```

Comment:
Nice test. Why is it even inside the test case for data_nodes? I believe, it should be a separate test_filesystem.py.